pokerole_irl/worlds_app/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+# Create your models here.
+class Setting(models.Model):
+    owner = models.ForeignKey(User, null=True, on_delete=models.SET_NULL)
+    members = models.ManyToManyField(User, related_name='+', blank=True)
+    name = models.CharField(max_length=100)
+    description = models.TextField(default="")
+
+
+    def __str__(self):
+        return self.name
+
+class Character(models.Model):
+    owner = models.ForeignKey(User, on_delete=models.CASCADE)
+    setting = models.ForeignKey(Setting, blank=True, null=True, on_delete=models.SET_NULL)
+    first_name = models.CharField(max_length=100)
+    last_name = models.CharField(max_length=100)
+    description = models.TextField(default="")
```

Comment:
Add more info for this please. This should probably contain all information a character would have in the game (stats, skills etc.).

Also, I think this model should have an "NPC"-boolean.

Reply:
Boolean, or should we use two separate models for NPC and PC? Character can be used as a parent class

Reply:
This may be the way, but let's consider this; What will the difference be between a PlayerCharacter and a NonPlayerCharacter?